src/components/conversation/handlers.js

```diff
@@ -560,9 +560,9 @@ function parseWSMessage(e) {
   }
 }
 
-function connectWebSocket() {
+function connectWebSocket(backend) {
```

Comment:
I think this whole file can maybe just be removed?

Reply:
Yeah pretty much everything in that folder except for the component can be removed.